playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -341,6 +341,17 @@
     - libvirt
     - run
 
+# we need to deploy the extra values for neutron so on the next task they are picked up
+- name: Deploy neutron overrides
+  import_tasks: component-install.yml
+  vars:
+    _cmpnt_template: neutron.yaml
+    _cmpnt_overrides: "{{ suse_osh_deploy_neutron_yaml_overrides }}"
+    _cmpnt_runcommand: "/bin/true" # we only want to create the template, not run anything
```

Comment:
This is smart, but maybe we can add some variable that would mean "Run the override file" is skipped?

Maybe if `_cmpnt_runcommand` is empty , some "when" clause would not run it?

I know the result is the same, but it might look better

Reply:
Yes, I didnt want to go further or anything with this as its kind of a workaround currently pending any further work upstream to divide that compute-kit?

So I wanted to make it work *now* fix it later if we need to go upstream to change it

Reply:
Well... it might have some other usage in theory, even if this particular script is splited upstream

Reply:
You can simply call config_template, like L175.